Traceback (most recent call last):
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/client.py", line 558, in async_execute
    await self.async_execute_cell(
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
let
    ts = [1.0, 1.3, 3.1, 4.0]
    xs = range(first(ts), last(ts); length=200)
    plt = plot(layout=(4,1),xlabel=L"x", ylims=[-0.1,1.1], ytick=[1], leg=:top)
    for k in 1:size(ts, 1)
        φₖ = hatfunc(ts, k)
        plot!(xs, φₖ.(xs); label="", subplot=k)
        scatter!(ts, φₖ.(ts); label="", subplot=k)
        annotate!(ts[k], 0.5, text("φ_$k", 10); subplot=k)
    end
    plt
end
------------------

LoadError: UndefVarError: @L_str not defined
in expression starting at In[1]:4

Stacktrace:
 [1] top-level scope
   @ :0
 [2] eval
   @ ./boot.jl:360 [inlined]
 [3] include_string(mapexpr::typeof(REPL.softscope), mod::Module, code::String, filename::String)
   @ Base ./loading.jl:1116
LoadError: LoadError: UndefVarError: @L_str not defined
in expression starting at In[1]:4

